cmd/bosun/web/search.go

```diff
@@ -63,6 +63,24 @@ func MetricsByTagPair(t miniprofiler.Timer, w http.ResponseWriter, r *http.Reque
 	return schedule.Search.MetricsByTagPair(tagk, tagv)
 }
 
+func MetricsByTagKey(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interface{}, error) {
```

Comment:
Perhaps a comment explaining what it returns with an example. map[string][]string could be anything